src/seq/bioseq.jl

```diff
@@ -1027,3 +1027,80 @@ end
 function enc64{A}(::BioSequence{A}, x)
     return UInt64(encode(A, convert(eltype(A), x)))
 end
+
+
+# Sequences to Matrix
+# -------------------
+
+"""
+    seqmatrix{A<:Alphabet}(vseq::Vector{BioSequence{A}}, major = :seq|:site)
```

Comment:
Inserting spaces around `=` is not so common in Julia. See `split` help for example.